packages/hawtio/src/plugins/camel/routes/RouteDiagram.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import { CamelContext } from '@hawtiosrc/plugins/camel/context'
+import ReactFlow, {
+  addEdge,
+  ConnectionLineType,
+  Edge,
+  Handle,
+  Node,
+  NodeProps,
+  NodeToolbar,
+  Position,
+  useEdgesState,
+  useNodesState,
+} from 'reactflow'
+import dagre from 'dagre'
+import { CamelNodeData, routeVizualisationService } from '@hawtiosrc/plugins/camel/routes/visualization-service'
+import 'reactflow/dist/style.css'
+import './routeDiagram.css'
+import { Flex, FlexItem, Page } from '@patternfly/react-core'
+
+const dagreGraph = new dagre.graphlib.Graph()
+dagreGraph.setDefaultEdgeLabel(() => ({}))
+
+const nodeWidth = 300
+const nodeHeight = 86
+
+export type ICamelNode = Node & {
```

Comment:
You don't need `I` prefix. Non only because it's not an interface, but also because we generally wouldn't use `I` prefixes other than service interfaces.